robocopkaka/bootcamp-29-project
# #158052789 Fix issues with material ui

#### What does this PR do?
use react-materialize components instead of material-ui ones
#### Description of Task to be completed?
* use react-materialize datepicker
* remove category drop-down since it's not really needed
#### How should this be manually tested?
* visit `https://rocky-meadow-55707.herokuapp.com`
* click on `Centers`
* click on any center
* page should load and you should see the detail of any center
#### Any background context you want to provide?
material-ui was causing problems in some components on Heroku
#### What are the relevant pivotal tracker stories?(if applicable)
#158052789
